compiler/modules/front.syntax/syntaxNode.bal

```diff
@@ -0,0 +1,831 @@
+type TerminalSyntaxNode TerminalSyntaxAstNode|FixedSyntaxNode|IdentifierSyntaxNode|StringLiteralSyntaxNode;
+type AstSyntaxNode TerminalSyntaxAstNode|NonTerminalSyntaxNode;
+type SubSyntaxNode NonTerminalSyntaxNode|TerminalSyntaxNode;
+type SyntaxNode SubSyntaxNode|RootSyntaxNode;
+
+const CLING = 0;
+const CLING_PREV = 1;
+const CLING_NEXT = 2;
+const NEWLINE_AFTER = 3;
+const NEWLINE_BEFORE = 4;
+const IGNORE_LITERAL_ESCAPE = 5;
+type OutputFlag CLING|CLING_PREV|CLING_NEXT|NEWLINE_AFTER|NEWLINE_BEFORE|IGNORE_LITERAL_ESCAPE;
```

Comment:
Flags are things that are bitwise or'ed together (i.e. with value 0x1, 0x2, 0x4, 0x8 etc).

i.e.

```
const CLING_PREV = 0x1;
const CLING_NEXT = 0x2;


const CLING = CLING_NEXT|CLING_PREV;
```

Then use like
```
if (flags & CLING_PREV) != 0 {
}
```

Reply:
fixed